### PR TITLE
Update addtask deletetask done parser

### LINKED ISSUE #46: Update Parser: AddTaskCommandParser

### LINKED ISSUE #48: Update Parser: DeleteTaskCommandParser

### LINKED ISSUE #49: Update Parser: DoneCommandParser

### PR DESCRIPTION
**Pull request by ljhgabe**
Closes #49, closes #48, closes #46

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -12,4 +12,8 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_DEADLINE = new Prefix("d/");
+    public static final Prefix PREFIX_PRIORITY = new Prefix("p/");
```
Comment (litone01) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java]: Priority and phone has the same prefix, may need to be careful later

**Review by litone01 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 1 above)
Comment (ljhgabe) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java]: Ok thanks!

File: src/main/java/seedu/address/logic/commands/DoneTaskCommand.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Task> lastShownList = model.getFilteredTaskList();
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_TASK_DISPLAYED_INDEX);
+        }
+
+        Task taskToMark = lastShownList.get(targetIndex.getZeroBased());
+        model.deleteTask(taskToMark);
```
Comment (litone01) [inline src/main/java/seedu/address/logic/commands/DoneTaskCommand.java]: I suppose this file is not updated fully yet

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to the task list. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + "[" + PREFIX_DEADLINE + "DEADLINE] "
+            + "[" + PREFIX_PRIORITY + "PRIORITY] "
+            + "[" + PREFIX_PRIORITY + "PRIORITY] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe "
```
Comment (litone01) [inline src/main/java/seedu/address/logic/commands/AddTaskCommand.java]: are we gonna change the prefix_phone and email all that?

Comment (ljhgabe) [reply src/main/java/seedu/address/logic/commands/AddTaskCommand.java]: Maybe i will hand over this part to you haha.

Comment (litone01) [reply src/main/java/seedu/address/logic/commands/AddTaskCommand.java]: oh ok haha

**Pull request merged**